### ISSUE TITLE
Meeting summary tool redesign - community input

### ISSUE THREAD

**Issue by VanCardui** [SingularityNET-Archive/SingularityNET-Archive#193]

**Comment by stephen-rowan** [comment]
Some consensus emerging that targeted one to one interviews may be preferable.
Approach participants and give them some time to consider summary tool improvements.

**Comment by oluayoola7d** [comment]
What is the update on this?
I suppose it will be discussed in the next meeting.

**Comment by VanCardui** [comment]
Agreed that we can no longer afford to fund this - instead, have used anecdotal input and input from the tool's issues log.
